ui/src/components/ripple-out/RippleOutContainer.js

```diff
@@ -1,344 +1,34 @@
-          >
-            <HStack paddingTop="40px" />
-            <Flex w="100%" flexDir="column" color="#ffffff">
-              <Text
-                textStyle="darker_grotesque_bold"
-                fontWeight="900"
-                fontSize={['4em', '14em']}
-                lineHeight={['0.25em', '0em']}
-              >
-                RIPPLE OUT
-              </Text>
-              <Text textStyle="darker_grotesque" fontSize={['3em', '12em']}>
-                CAMPAIGN
-              </Text>
-            </Flex>
-            <HStack
-              alignItems="center"
-              w="100%"
-              paddingBottom="20px"
-              spacing={2}
-            >
-              <Text
-                style={navStyleMobile}
-                onClick={() =>
-                  storyRef.current.scrollIntoView({
-                    behavior: 'smooth',
-                    block: 'start',
-                    inline: 'nearest',
-                  })
-                }
-              >
-                THE STORY
-              </Text>
-              <Box style={navDotStyleMobile} />
-              <Text
-                style={navStyleMobile}
-                onClick={() =>
-                  stepsRef.current.scrollIntoView({
-                    behavior: 'smooth',
-                    block: 'start',
-                    inline: 'nearest',
-                  })
-                }
-              >
-                THE STEPS
-              </Text>
-              <Box style={navDotStyleMobile} />
-              <Text
-                style={navStyleMobile}
-                onClick={() =>
-                  spaceRef.current.scrollIntoView({
-                    behavior: 'smooth',
-                    block: 'start',
-                    inline: 'nearest',
-                  })
-                }
-              >
-                THE SPACE
-              </Text>
-              <Box style={navDotStyleMobile} />
-              <Text
-                style={navStyleMobile}
-                onClick={() =>
-                  supportRef.current.scrollIntoView({
-                    behavior: 'smooth',
-                    block: 'start',
-                    inline: 'nearest',
-                  })
-                }
-              >
-                THE SUPPORT
-              </Text>
-            </HStack>
-          </Center>
-        </Container>
-      </Box>
-      <Flex
-        minH="100%"
-        flexDir="column"
-        bgColor="#FFFBEF"
-        py={['1.8em', '4em']}
-        color={fontColor}
-        ref={storyRef}
-      >
-        <Container
-          maxW={['container.xl']}
-          h="100%"
-          textStyle="darker_grotesque"
-        >
-          <Flex flexDir="column" gap={9}>
-            <RippleOutSubHero
-              headerFontSize={headerFontSize}
-              bodyFontSize={bodyFontSize}
-            />
-            <ProgressSection bodyFontSize={bodyFontSize} />
+      {/* <RippleOutHeroSection /> */}
```

Comment:
just delete all these comments... lol and we also don't need this file anymore? why still keep it and only have Story section?